### DISCUSSION TITLE
init class in react

### DISCUSSION THREAD

**Discussion by krishnaTORQUE** [twbs/bootstrap#33895] [General]
I am using react & how can I init class of bootsrap `newbootsrap` ?

below example does not working
```
import 'bootstrap/dist/js/bootstrap.bundle.min';

 const modalElm: any = new (window as any).bootstrap.Modal(
      document.querySelector('.modal'),
      {
        backdrop: 'static',
        keyboard: false,
        show: true,
      }
    );

modalElm.show();
```

Error
`window.bootstrap is undefined`

**Comment by ImAbhishekTomar** [comment]
for init  javascript i am also searching but for css try this.

https://dev.to/danwalsh/getting-started-with-the-new-bootstrap-5-beta-and-nextjs-51am
